test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxJunctionActivity.kt

```diff
@@ -0,0 +1,260 @@
+            val or = Point.fromLngLat(139.7745686, 35.677573)
+            val de = Point.fromLngLat(139.784915, 35.680960)
+            findRoute(or, de)
+        }
+        return false
+    }
+
+    private fun init() {
+        initNavigation()
+        initStyle()
+        routeLine.initialize(binding.mapView, mapboxNavigation)
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun initNavigation() {
+        mapboxNavigation = MapboxNavigation(
+            NavigationOptions.Builder(this)
+                .accessToken(getMapboxAccessTokenFromResources())
+                .locationEngine(ReplayLocationEngine(mapboxReplayer))
+                .build()
+        )
+        junctionApi = MapboxJunctionApi(getMapboxRouteAccessToken(this))
+        mapboxNavigation.startTripSession()
+        mapboxReplayer.pushRealLocation(this, 0.0)
+        mapboxReplayer.play()
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun initStyle() {
+        mapboxMap.loadStyleUri(
+            MAPBOX_STREETS
+        ) {
+            getGesturePlugin().addOnMapLongClickListener(this)
+        }
+    }
+
+    private fun findRoute(origin: Point, destination: Point) {
+        mapboxNavigation.requestRoutes(
+            RouteOptions.builder()
+                .applyDefaultParams()
+                .accessToken(Objects.requireNonNull(getMapboxRouteAccessToken(this)))
+                .coordinates(listOf(origin, destination))
+                .alternatives(true)
+                .build()
+        )
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun vibrate() {
+        val vibrator = getSystemService(VIBRATOR_SERVICE) as Vibrator
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            vibrator.vibrate(VibrationEffect.createOneShot(100L, DEFAULT_AMPLITUDE))
+        } else {
+            vibrator.vibrate(100L)
+        }
+    }
+
+    private fun getMapCamera(): CameraAnimationsPlugin {
+        return binding.mapView.getCameraAnimationsPlugin()
+    }
+
+    private fun getGesturePlugin(): GesturesPlugin {
+        return binding.mapView.getGesturesPlugin()
+    }
+
+    private fun getMapboxAccessTokenFromResources(): String? {
+        return getMapboxAccessToken(this)
+    }
+
+    private fun updateCamera(location: Location) {
+        val mapAnimationOptionsBuilder = MapAnimationOptions.Builder()
+        mapAnimationOptionsBuilder.duration(1500L)
+        mapCamera.easeTo(
+            CameraOptions.Builder()
+                .center(Point.fromLngLat(location.longitude, location.latitude))
+                .bearing(location.bearing.toDouble())
+                .pitch(45.0)
+                .zoom(17.0)
+                .padding(EdgeInsets(1000.0, 0.0, 0.0, 0.0))
+                .build(),
+            mapAnimationOptionsBuilder.build()
+        )
+    }
+
+    /**
+     * Uses a specific access token required for the route request to send junctions in the response.
```

Comment:
```suggestion
     * Uses a specific access token required for the route request to send signboards in the response.
```

Reply:
This is correct and should not be `signboard` since it is `MapboxJunctionActivity`